osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -22,17 +22,19 @@ public WorkingBeatmap Beatmap
             {
                 return beatmap;
             }
-
             set
             {
-                if (beatmap == value)
+                if (beatmap == value && beatmap != null)
                     return;
-
                 beatmap = value;
 
                 Schedule(() =>
                 {
-                    Background newBackground = new BeatmapBackground(beatmap);
+                    Background newBackground;
+                    if (beatmap == null)

```

Comment:
You can still prohibit things or add special behavior by checking `beatmap == Default`. In the majority of cases you don't need that, though. The beatmap could have the osu! theme as song, and would be built-in into the executable anyway, so you also avoid the logic needed to play the theme song when no beatmap is selected (which can become surprisingly annoying when all transitioning to beatmap-based audio need to be seamless).